components/assetcard/stories/assetcard.stories.js

```diff
@@ -161,3 +160,6 @@ DropTarget.args = {
 	isDropTarget: true,
 	isSelected: true,
 };
+
+export const Express = Template.bind({});
```

Comment:
This PR is adding a lot of Express stories. Won't these all need to be removed when we move to S2? It's reasonable to expect that users can use the control panel to see Express, and we can add Chromatic coverage for Express to the default story using `isChromatic()`.

Reply:
It's adding about 8 I believe - I can check on that number - and yes it's expanding coverage but it's pretty important because we're going to start removing Express for S2 and we want to ensure no regressions as we go through that process. Express is used in a pretty high visibility project. I think it's worth it

Reply:
I'll also note, it's a theme that impacts every state and variant so adding it as just another item in the collection would make snapshots unmanageable to debug in my opinion.

Reply:
Ok. In that case, can we add the variants (wrapped with `isChromatic()`) to the Express story so that we get VRT coverage of the variants with Express turned on?

Reply:
No need - the express stories are leveraging the groupings which include isChromatic() so it's inheriting the full sets of tests where they are available.

Reply:
Asset card doesn't have groupings available which is why you're not seeing it here but check the [ActionButtons](https://github.com/adobe/spectrum-css/pull/2398/files#diff-b965736461e30accd067d0f9fe09aa0e60ac58b4910b6dfb39c194bbb5200f5a)